src/vm/functions/database.rs

```diff
@@ -43,7 +42,7 @@ pub fn special_fetch_variable(args: &[SymbolicExpression],
     let var_name = args[0].match_atom()
         .ok_or(UncheckedError::ExpectedVariableName)?;
 
-    env.global_context.database.lookup_variable(&env.contract_context.name, var_name)
+    env.global_context.database.lookup_variable(&&env.contract_context.name, var_name)
```

Comment:
is `&&` a sign that we're doing something wrong?

Reply:
I'm pretty sure that this can be safely changed to `&`, but I want to make sure that this wasn't a change in rustc `1.35`

Reply:
Updated